content/post/cookie/introduction/index.md

```diff
@@ -0,0 +1,202 @@
+  readers.
+- [Principles][]: Some general principles from the [NSLS-II][] guide.
+- [Patterns][]: Recipes for common situations. Three pages are there now;
+  [including data][], [backports][], and [exports][].
+
+From the original [Scikit-HEP dev pages][], a lot was added:
+
+- Brand new guide page on documentation, along with new sp-repo-review checks to
+  help with readthedocs.
+- A compiled projects page! Finally! With [scikit-build-core][],
+  [meson-python][], and [maturin][]. The page shows real outputs from the
+  [cookiecutter][], kept in sync with [cog][] (a huge benefit of using a single
+  repo for all three components!)
+- Big update to [GHA CI page][gha_basic] including a section on Composite
+  Actions given at the Dev summit.
+- CLI entry points are now included.
+- Python 3.12 support added, Python 3.7 dropped.
+- New [sp-repo-review][] badges throughout (more on that later!)
+- Updates for [Ruff][]'s move and support for requires-python.
+- Lots of additions for GitHub Actions.
+
+The infrastructure was updated too:
+
+- Using latest Jekyll (version 4) and latest Just the Docs theme. More colorful
+  callout boxes. Plugins are used now.
+- Live PR preview (provided by probably the world’s first readthedocs Jekyll
+  build!). Developed with the Zarr developers during the summit.
+- Better advertising for [cookie][] and [sp-repo-review][] on the index page(s).
+- Auto bump and auto-sync CI jobs.
+
+## Cookie
+
+We also did something I've wanted to do for a long time: the guide, the
+cookiecutter template, and the checks are all in a single repo! The repo is
+[scientific-python/cookie][cookie], which is the moved `scikit-hep/cookie` (the
+old URL for cookiecutter still works!). A lot of work went into the
+cookiecutter, too!
+
+- Generalized defaults. We still have special integration if someone sets the
+  org to `scikit-hep`; the same integration can be offered to other orgs.
+- All custom hooks removed; standard jinja now used throughout templates. Using
+  cookiecutter computed variables idiom too. Windows still fully supported and
+  tested. Adding new choices is much easier now.
+- Added cruft (a cookiecutter updater) testing.
+- Dual-supporting [copier][] now too, a [cookiecutter][] replacement with a huge
+  CLI improvement (and also supports updating). Might be the first project to
+  support both at the same time. CI (or [nox][] locally) checks to ensure
+  generation is identical. Much better interface with copier, including
+  validation, descriptive text, arrow selection, etc.
+- Improved CLI experience even if using cookiecutter (like no longer requesting
+  current year).
+- Reworked docs template.
+- Support for cookiecutter 2.2 pretty descriptions (added about four hours after
+  cookiecutter 2.2.0 was released).
+- GitLab CI support when not targeting github.com URLs (added by Giordon Stark).
+
+## Repo-review
+
+I wrote an [introduction post](/post/repo-review/) about this one!
```

Comment:
This link currently fails the link checker.

Reply:
Oops, copy-and past from my blog where it does work.

Do we have a photo we can add?

Reply:
Yes, I will get you add  a couple of photos soon (I am at the airport now).

Reply:
@henryiii Would you mind cross posting the repo-review intro post here as well (i.e., `/posts/scientific-python/repo-review/` in this repository). Maybe you could do that in a separate PR, which we could merge and then update this:
```
<!--fix
I wrote an [introduction post](/posts/scientific-python/repo-review/) about this one!
-->
```

If you prefer not cross-posting, then we should fix the above with the URL of the existing post.

Reply:
I'd need tabs implemented to cross-post. https://gitlab.com/iscinumpy/iscinumpy.gitlab.io/-/blob/master/layouts/shortcodes/tabs.html & https://gitlab.com/iscinumpy/iscinumpy.gitlab.io/-/blob/master/layouts/shortcodes/tab.html.